src/primal_dual_hybrid_gradient.jl

```diff
@@ -144,12 +144,18 @@ struct PdhgParameters
   pock_chambolle_alpha::Union{Float64,Nothing}
 
   """
-  Used to bias the computation of the primal/dual balancing parameter
-  primal_weight. Must be positive. A value of 1 balances primal and dual
-  equally.
+  Used to bias the initial value of the primal/dual balancing parameter
+  primal_weight. Must be positive. See scale_invariant_initial_primal_weight.
```

Comment:
(same comment as for mirror_prox.jl): Perhaps "See" -> "See also"? To me that makes it clearer that it is pointing to a source of additional information about the parameter.